bin/happo-ci

```diff
@@ -82,6 +86,11 @@ run-happo "$CURRENT_SHA" true
 if ! "$HAPPO_COMMAND" has-report "$PREVIOUS_SHA"; then
   echo "No previous report found for ${PREVIOUS_SHA}. Generating one..."
   run-happo "$PREVIOUS_SHA" false
+  
+  # Restore git and node_modules to their original state, for anything that
+  # might be run after this script.
+  git checkout "$CURRENT_SHA"
+  run-install
```

Comment:
This is a little unfortunate as it does take a little while. But I get why it's needed. 👍